### ISSUE TITLE
api: move pipettes away before starting heater-shaker shake

### ISSUE THREAD

**Issue by sanni-t** [Opentrons/opentrons#11209]
#9915 & #10542 restricts moving the pipette to slots adjacent to the heater-shaker while the h/s is shaking.
This ticket is about taking care of the inverse case of it- shaking while the pipette is parked *in/over a well* of labware adjacent to the shaking h/s. 

Implementation:
- Check if the pipettes' last known location was on or EWNS of the heater-shaker in question OR is unknown.
- If either of those are true, implement a method to retract both pipettes along the z-axis before starting the shake.

Tracks both engine & PAPIv2 side of work.
